cucumber.js

```diff
@@ -1,10 +1,8 @@
 module.exports = {
-    default: '--require features --tags ~@stress --tags ~@todo',
-    verify: '--require features --tags ~@todo --tags ~@bug --tags ~@stress -f progress',
-    jenkins: '--require features --tags ~@todo --tags ~@bug --tags ~@stress --tags ~@options -f progress',
-    bugs: '--require features --tags @bug',
```

Comment:
Fully agree on the clean-up.
I tried to use todo as `missing feature` and bug as `broken feature`. I don't think that this separation serves much of a purpose. I am missing a complete overview if there can be important distinctions  in the code-base but I agree that it looks like a good candidate to clean-up.